pkg/intel/metadata/bg/bgbootpolicy/bpmh.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017-2023 the LinuxBoot Authors. All rights reserved
```

Comment:
This here is the same code as in `cbnt/cbntbootpolicy/bpmh_manifestcodegen.go` - can we get rid of that, and also other duplication?